### DISCUSSION TITLE
a visual UI bug in the iOS app

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18235] [Mobile] [answered]
[this](https://user-images.githubusercontent.com/63432932/173015766-2e45919f-dd1b-4adf-875f-245f6acfdb4f.MOV) happens when I hold on "That's all for now".

**Comment by brianlovin** [comment, marked as answer]
Thank you! Flagging to our iOS team :)

**Reply by 3m4r5** [reply]
thanks
